CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+- `calcite-popover` - `max-width` has been removed. Content may need width set.
```

Comment:
I'd remove this second item, and then move the one under breaking change to a new `Updated` heading.

Reply:
Cool.

Reply:
Done.